README.md

```diff
@@ -167,13 +167,114 @@ wrapping a component you intend to style, you'll need to make sure you accept
 the `className` as a prop and apply it to where you want the styles applied in
 your custom component (normally the root element).
 
-##### ...styles
+#### ...styles
 
 The `glamorousComponentFactory` accepts any number of style object arguments.
 These can be style objects or functions which are invoked with `props` on every
 render and return style objects. To learn more about what these style objects
 can look like, please take a look at the [`glamor`][glamor] documentation.
 
+Style objects can affect pseudo-classes and pseduo-elements, complex CSS
```

Comment:
This is perfect! I want to avoid the README getting too long (maybe someday we'll do a docs website or something). For now, could we wrap this in:

```
<details>
<summary>Examples</summary>

/* All of your stuff here */
</details>
```